app/views/tickets/index.html.erb

```diff
@@ -44,7 +44,7 @@
 				<% if !ticket.can_see_reason?(CurrentUser.user) %>
 					<td><span style="cursor:help;" class="redtext" title="Due to privacy concerns, this information is confidential">Confidential</span></td>
 				<% else %>
-					<td class="ticket-subject" title="<%= h(truncate(strip_dtext(ticket.reason), length: 200)) %>"><%= link_to h(strip_dtext(truncate(ticket.subject, length: 200))), action: "show", id: ticket.id %></td>
+					<td class="ticket-subject" title="<%= h(truncate(strip_dtext(ticket.reason), length: 200)) %>"><%= link_to h(truncate(strip_dtext(ticket.subject), length: 200)), action: "show", id: ticket.id %></td>
```

Comment:
I'd ask that you drop this commit for right now and I will investigate and address it further. Subjects probably shouldn't be allowed html, or dtext for that matter.